src/utils/match.ts

```diff
@@ -0,0 +1 @@
+export const isImageFormat = (str: string) => new RegExp(".*(?:jpg|gif|png|jpeg|svg|webp)").test(str)
```

Comment:
Couple things
- Can we change the name of the function? I'm not sure what "isImageFormat" is telling me (by name); maybe something like "isImageFilename" or something
- The regex should be constructed ones
- It's clearer to say export function instead of export const that is a function

Reply:
What do you mean by constructed regex?

Reply:
The current version of this function creates a new RegExp instance (with the same pattern). That creation/construction/instantiation should occur once (since the pattern is static)